src/includes/Helper.php

```diff
@@ -156,19 +156,27 @@ public static function getBeansPages()
 
     public static function getCurrentPage()
     {
-        $pages = [
-            wc_get_cart_url()                               => 'cart',
-            wc_get_checkout_url()                           => 'cart',
-            # DON'T TOUCH: This helps to show the redeem button on the checkout page
-            wc_get_page_permalink('shop')                   => 'product',
-            wc_get_page_permalink('myaccount')              => 'login',
-            get_permalink(Helper::getConfig('liana_page'))  => 'reward',
-            get_permalink(Helper::getConfig('bamboo_page')) => 'referral',
-        ];
-
-        $current_page = esc_url(home_url($_SERVER['REQUEST_URI']));
-        $current_page = explode("?", $current_page)[0];
-        return isset($pages[$current_page]) ? $pages[$current_page] : '';
+        if (is_front_page()) {
+            $result = 'home';
```

Comment:
Use early return. It makes the code more ready 
`if (is_front_page()) return "home"`